README.md

```diff
@@ -508,6 +508,26 @@ Generates the following response payload:
 }
 ```
 
+### `Boom.locked([message], [data])`
+
+Returns a 423 Locked error where:
+- `message` - optional message.
+- `data` - optional additional error data.
+
+```js
+Boom.locked('this account has been locked and it is your fault');
```

Comment:
Agreed. Also not sure "account" is strictly necessary either. Why not just:

- The resource that is being accessed is locked
- This resource has been locked